_pages/home/spartacus-faq.md

```diff
@@ -3,131 +3,140 @@ title: Spartacus FAQ
 
-### Does this mean the Accelerators will be deprecated? 
+### Does this mean the Accelerators will be deprecated?
 
 Eventually, but that is many years out. Our goal is to move all customers to Spartacus, or at least JavaScript-based storefronts. All current development efforts are focused on improving Spartacus. For the moment, Accelerator templates are receiving critical and blocker bug or security fixes only. At the very least, Accelerator templates won’t be deprecated until Spartacus achieves feature parity. Spartacus is the strategic way forward for maintaining a customizable storefront with SAP Commerce Cloud. Please see the article “Choosing Which Storefront to Use for Your SAP Commerce Cloud Solution” for more information on this topic. 
 
-### If I’m currently using an SAP Commerce Cloud Accelerator, how do I migrate to a Spartacus-based storefront? 
+### If I’m currently using an SAP Commerce Cloud Accelerator, how do I migrate to a Spartacus-based storefront?
 
 Spartacus is a complete paradigm shift from the Accelerators in terms of technologies and architecture (template vs. libraries, headless vs. embedded, JSP vs Angular). As such there is no direct way to migrate from an Accelerator-based storefront to one that uses the Spartacus libraries. Given that it would take time to move an Accelerator-based storefront to Spartacus, we recommend a piece-by-piece approach, where parts of Spartacus functionality are used alongside Accelerator-based storefront, with seamless switching between the two. Another consideration is that custom functionality requires APIs for Spartacus. On the other hand, developing a JavaScript-based front-end is a much faster process than with Accelerator. 
 
-### How long does it take to get a Spartacus-based storefront up and running? 
+### How long does it take to get a Spartacus-based storefront up and running?
 
 Assuming a vanilla SAP Commerce Cloud B2C backend is up and running and configured to accept OCC API calls, a developer can get create a Spartacus-based storefront within 10 minutes. See [Building the Spartacus Storefront from Libraries]({{ site.baseurl }}{% link _pages/install/building-the-spartacus-storefront-from-libraries.md %}).
 
-### Is there a demo of a Spartacus-based storefront available? 
+### Is there a demo of a Spartacus-based storefront available?
 
 Try out our continuous-integration server [here](https://sap.github.io/cloud-commerce-spartacus-storefront). As of this writing (April 19, 2019), note that the backend server used by this sample site is not up to date with latest 1905, due to the imminent release of 1905 late May. The server will be updated shortly after release.  
```

Comment:
outdated

Reply:
All text after the link has been removed.